<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstallWizard</class>
 <widget class="QWizard" name="InstallWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;安装向导&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWizardPage" name="Page1">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ff5500;&quot;&gt;用户安全协议&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;请务必仔细阅读和理解此金山软件最终用户许可协议（“本《协议》”）中规定的所有权利和限制。在安装时，您需要仔细阅读并决定接受或不接受本《协议》的条款。除非或直至您接受本《协议》的条款，否则本“软件”不得安装在您的计算机上。 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;   作为参考，您现在就可以从本页起打印出本《协议》的文本，或者参阅本“软件”“帮助”文件中本《协议》的副本。 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;   本《协议》是您与金山软件之间有关本“软件”的法律协议。本“软件”包括随附的计算机软件，并可能包括相关文档印刷材料。您一旦安装本“软件”，即表示您同意接受本《协议》各项条款的约束。如您不同意本《协议》中的条款，您则不可以安装或使用本“软件”。 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  本“软件”受著作权法及国际著作权条约和其它知识产权法和条约的保护。本“软件”权利只许可使用，而不出售。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  &lt;span style=&quot; font-weight:600;&quot;&gt;一．金山软件将本“软件”在中华人民共和国（“中国”）大陆地区（除香港特别行政区、澳门特别行政区、台湾地区外的中国境内）的非专有的使用权授予您。 您可以：&lt;/span&gt; &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  1.在一台个人所有计算机上安装、使用、显示、运行（“运行”）本“软件” 的一份副本。本“软件”为个人版软件，仅供个人所有计算机使用，不得用于法人或其他组织（包括但不限于政府机关、公司、企事业单位、其他组织等；无论该组织是否为经济性组织；无论该组织的使用是否构成商业目的使用）所有计算机。如若个人所有计算机长期固定为法人或其他组织服务，则将被视为“法人或其他组织所有计算机”，无法享受本授权。任何超出上述授权范围的使用均被视为非法复制的盗版行为，金山软件保留权利要求相关责任人承担相应的法律责任包括但不限于民事责任、行政责任、刑事责任。 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  2.为了防止副本损坏而制作备份复制品。这些备份复制品不得通过任何方式提供给他人使用，并在您丧失该合法副本的所有权时，负责将备份复制品销毁。 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  3.为了把本“软件”用于实际的计算机应用环境或者改进其功能、性能而进行必要的修改；但是，除本《协议》另有约定外，未经金山软件许可，不得向任何第三方提供修改后的软件。 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;二．您保证：&lt;/span&gt; &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  1.不在本《协议》规定的条款之外，使用、复制、修改或转让本“软件”或其中的任一部分。 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  2.只在一台个人所有计算机上使用本“软件”；一份软件许可不得在不同的计算机上共同或同时使用。 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  3.只在以下前提完全满足的情况下，将本“软件”用于多用户环境或网络系统上：&lt;br /&gt;  本“软件”明文许可用于多用户环境或网络系统上；&lt;br /&gt;  使用本“软件”的每一节点及终端都已购买使用许可。 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  4.不得对本“软件”进行反向工程、反向编译或反汇编；不得试图进行任何获得本“软件”原代码的访问或行为。 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  5.不得发行、出租、信息网络传播、翻译本“软件”。 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  6.在本“软件”的所有副本上包含所有的版权标识。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="agreecheck">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>我已阅读用户保护协议，并同意此协议</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="Page2">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>用户</string>
    </property>
    <widget class="QCheckBox" name="currentUser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>当前用户</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="allUser">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>所有用户</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>兼容性</string>
    </property>
    <widget class="QCheckBox" name="checkBoxDoc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>关键Doc、XSL、PPT</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPdf">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>关联PDF格式</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPng">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>关联PNG、JEPG图片格式</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxEpue">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>关联EPUBE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="installPath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Install">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>安装位置</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="Page3">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>6</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>6</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>安装进度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>安装文件列表：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ProgressValue">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outputFilename">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="Page4">
   <widget class="QComboBox" name="comboBoxChoose">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>11</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>序列号</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>账号登录</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>激活方式：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="skipcheck">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>跳过</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>339</width>
       <height>139</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="orderLineEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CACEE-EEXSC-ESXDF-RFDGFG</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="accountLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>账号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="activationButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>激活</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>agreecheck</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
